workflow/rules/trimming.smk

```diff
@@ -71,33 +71,40 @@ rule ptrimmer_se:
         "results/trimmed/adapters/{sample}/{unit}.single_R1.fastq.gz"
     output:
         "results/trimmed/primers/{sample}/{unit}.single.fastq.gz",
-    log:
-        "logs/ptrimmer/{sample}-{unit}.log"
     params:
         primers=config["primers"]["trimming"]["primers"],
-
+        output_dir=lambda wc, output: os.path.dirname(output)
+    log:
+        "logs/ptrimmer/{sample}-{unit}.log"
     conda:
         "../envs/ptrimmer.yaml"
     shell:
-        "ptrimmer -f {input} -a {params.primers} -o ./ > {log}"
-        "gzip -c results/trimmed/adapters/{wildcards.sample}/{wildcards.unit}.single_trim_R1.fq > {output}"
+        "ptrimmer -s single -f {input} -a {params.primers} -o {params.output_dir} &> {log} && "
+        "gzip -c results/trimmed/primers/{wildcards.sample}/{wildcards.unit}.single_trim_R1.fq > {output} && "
+        "rm gzip -c results/trimmed/primers/{wildcards.sample}/{wildcards.unit}.single_trim_R1.fq"
```

Comment:
Is this intentional? Looks like a copy-paste error.

Reply:
Indeed. Should be fixed now.